gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -155,10 +166,104 @@ def import_spliceai_vcf(path: str) -> hl.Table:
     logger.info("Getting the max SpliceAI score for each variant across genes...")
     ht = ht.collect_by_key()
     ht = ht.select(splice_ai=hl.struct(ds_max=hl.max(ht.values.ds_max)))
-
+    ht = ht.annotate_globals(spliceai_version="v1.3")
     return ht
 
 
+def create_revel_grch38_ht() -> hl.Table:
+    """
+    Create a Hail Table with REVEL scores for GRCh38.
+
+    .. note::
+    Starting with gnomAD v4, we use REVEL scores for only MANE Select and
+    canonical transcripts. Even when a variant falls on multiple MANE/canonical
+    transcripts of different genes, the scores are equal.
+    REVEL scores were downloaded from:
+       https://rothsj06.dmz.hpc.mssm.edu/revel-v1.3_all_chromosomes.zip
+       size ~648M, ~82,100,677 variants
+    REVEL's Ensembl ID is not from Ensembl 105, so we filter to transcripts
+    that are in Ensembl 105. The Ensembl 105 ID file was downloaded from Ensembl
+    105 archive. It contains the following columns:
+       Transcript stable ID, Ensembl Canonical, MANE Select
+    This deprecates the `has_duplicate` field present in gnomAD v3.
+
+    :return: Hail Table with REVEL scores for GRCh38.
+    """
+    revel_csv = "gs://gnomad-insilico/revel/revel-v1.3_all_chromosomes_with_transcript_ids.csv.bgz"
+    ensembl_path = "gs://gnomad-insilico/ensembl/ensembl105id.grch38.tsv.bgz"
+
+    ht = hl.import_table(
+        revel_csv,
+        delimiter=",",
+        min_partitions=1000,
+        types={"grch38_pos": hl.tstr, "REVEL": hl.tfloat64},
+    )
+
+    logger.info("Annotating REVEL table...")
+    ht = ht.drop("hg19_pos", "aaref", "aaalt")
+    # drop variants that have no position in GRCh38 when lifted over from GRCh37
+    ht = ht.filter(ht.grch38_pos.contains("."), keep=False)
+    ht = ht.transmute(chr="chr" + ht.chr)
+    ht = ht.annotate(
+        locus=hl.locus(ht.chr, hl.int(ht.grch38_pos), reference_genome="GRCh38"),
+        alleles=hl.array([ht.ref, ht.alt]),
+        Transcript_stable_ID=ht.Ensembl_transcriptid.strip().split(";"),
+    )
+    ht = ht.select("locus", "alleles", "REVEL", "Transcript_stable_ID")
```

Comment:
```suggestion
    ht = ht.select(
        locus=hl.locus(ht.chr, hl.int(ht.grch38_pos), reference_genome="GRCh38"),
        alleles=hl.array([ht.ref, ht.alt]),
        REVEL=ht.REVEL,
        Transcript_stable_ID=ht.Ensembl_transcriptid.strip().split(";"),
    )
```
You can treat a select like an annotate which is cool, this is just another way of doing what you did but figured I'd demonstrate it here.